Platform/QemuBoardPkg/BoardConfig.py

```diff
@@ -67,6 +67,7 @@ def __init__(self, *args, **kwargs):
 
         self.ENABLE_SMBIOS            = 1
         self.ENABLE_SBL_SETUP         = 0
+        self.ENABLE_UPL_HANDOFF_FDT   = 0
```

Comment:
Should be TRUE or 1 to enable?

Reply:
SBL supports both HOB and FDT. Keeping it 0 to be back-compatible to pass CI test. Adding this just for user easily enable FDT support.